checkbox-provider-kivu/bin/compare_power_consumption.py

```diff
@@ -34,27 +45,27 @@ def load_energy(fname: str ) -> tuple[float, dict[str, float]]:
     no_hwacc_energy_file = sys.argv[2]
     min_gain = float(sys.argv[3])
 
+    # data format
+    # Example: (600.097183, {'psys': 9976.218114, 'package-0': 3658.829745, 'core': 825.515099, 'uncore': 318.009806})
```

Comment:
if you have the realworld numbers, why not create a unit tests out of them?

Reply:
> if you have the realworld numbers, why not create a unit tests out of them?

no, they are not real world numbers, just an example showing how data will look like